models/players.js

```diff
@@ -42,6 +42,23 @@ var returnPlayers = function (players, rb_team_id, res, league){
   }
 }
 
+var returnGoatPlayers = function (players, list_id, rb_team_id, res, league){
```

Comment:
This is pretty CRY, why not just use the returnPlayers function and differentiate between lists in that one?